frontend/components/tree/js/tree.component.ts

```diff
@@ -124,10 +115,7 @@ class TreeController implements ng.IController {
 
 			} else if (event.type === this.EventService.EVENT.VIEWER.BACKGROUND_SELECTED) {
 				this.TreeService.clearCurrentlySelected();
-				if (this.currentFilterItemSelected !== null) {
-					this.currentFilterItemSelected.class = "";
-					this.currentFilterItemSelected = null;
-				}
+				this.nodes.forEach((n) => n.class = "");
```

Comment:
What is this used for? (I know you probably didn't write it)

From a bit of crawling through the code it seems to be either set as "treeNodeSelected" or "unselectedFilterItem" ? 
Is there a better way to represent this? I feel like this should be a boolean or something